certregistry/cert.go

```diff
@@ -0,0 +1,80 @@
+package certregistry
```

Comment:
I belief that all of this code should be part of the secrets module instead, maybe as submodule.